org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xsi/XSISchemaModel.java

```diff
@@ -159,36 +155,42 @@ private static void createCompletionItem(String attrName, boolean canSupportSnip
 		response.addCompletionItem(item);
 	}
 
-	public static void computeValueCompletionResponses(ICompletionRequest request, 
-			ICompletionResponse response, DOMDocument document) throws BadLocationException {
+	public static void computeValueCompletionResponses(ICompletionRequest request, ICompletionResponse response,
+			DOMDocument document) throws BadLocationException {
 		Range editRange = request.getReplaceRange();
 		int offset = document.offsetAt(editRange.getStart());
 		DOMNode nodeAtOffset = request.getNode();
-		
+
 		String actualPrefix = document.getSchemaInstancePrefix();
 		DOMAttr attrAtOffset = nodeAtOffset.findAttrAt(offset);
 		if (attrAtOffset == null) {
 			return;
 		}
 		String attrName = attrAtOffset.getName();
-		
-		if(attrName != null) {
-			if(actualPrefix != null && attrName.equals(actualPrefix + ":nil")) { // Value completion for 'nil' attribute
+
+		if (attrName != null) {
+			if (actualPrefix != null && attrName.equals(actualPrefix + ":nil")) { // Value completion for 'nil'
```

Comment:
Put the comment on the line before the `if` statement

Reply:
fixed